<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::PromptDialog</class>
 <widget class="QDialog" name="ns::PromptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetInner {
	border: 1px solid #4D5F6B;
	border-radius: 4px;
}

/* caption panel style */
#frameCaption {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #0078c7, stop: 1.0 #016aae);
	border-top: 1px solid #58A8DB;
	border-right: 1px solid #58A8DB;
	border-left: 1px solid #58A8DB;
	border-bottom: 1px solid #58A8DB;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

#frameCaption #labelCaptionLogo {
	background-image: url(:/images/logo.png);
	background-repeat: none;
	background-position: center;
	max-width: 32px;
	min-width: 32px;
	max-height: 32px;
	min-height: 32px;
}

#frameCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #ffffff;
}

#frameCaption #frameSystemMenu {
	border: 0px;
}

#frameCaption #frameSystemMenu #sysPanelContainer {
	border: 0px;
}

#frameCaption #pushButtonClose {
	border-image: url(:/images/caption_close.png);
}

#frameCaption #pushButtonClose:hover {
	border-image: url(:/images/caption_close_hover.png);
}

#frameCaption #pushButtonClose:pressed {
	border-image: url(:/images/caption_close_pressed.png);
}

#panel {
	/*border: 1px solid #BDE0F4;*/
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #9BD0F0, stop: 0.2 #EFF8FF);
}

#labelContainer #labelText {
	font-size: 20px;
}

#buttonContainer {
	background: #E1F1FE;
	border-top: 1px solid #FCFDFF;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

#pushButtonOK {
	border-image: url(:/images/prompt_ok.png) 0 3 0 3;
	color: #FFFFFF;
	border-top: 0 transparent;
	border-right: 3 transparent;
	border-bottom: 0 transparent;
	border-left: 3 transparent;
}

#pushButtonOK:hover {
	border-image: url(:/images/prompt_ok_hover.png) 0 3 0 3;
}

#pushButtonOK:pressed {
	border-image: url(:/images/prompt_ok_pressed.png) 0 3 0 3;
}

#pushButtonYes {
	color: #FFFFFF;
	border-image: url(:/images/prompt_ok.png) 0 3 0 3;
	border-top: 0 transparent;
	border-right: 3 transparent;
	border-bottom: 0 transparent;
	border-left: 3 transparent;
}

#pushButtonYes:hover {
	border-image: url(:/images/prompt_ok_hover.png) 0 3 0 3;
}

#pushButtonYes:pressed {
	border-image: url(:/images/prompt_ok_pressed.png) 0 3 0 3;
}

#pushButtonNo {
	border-image: url(:/images/prompt_cancel.png) 0 3 0 3;
	border-top: 0 transparent;
	border-right: 3 transparent;
	border-bottom: 0 transparent;
	border-left: 3 transparent;
}

#pushButtonNo:hover {
	border-image: url(:/images/prompt_cancel_hover.png) 0 3 0 3;
}

#pushButtonNo:pressed {
	border-image: url(:/images/prompt_cancel_pressed.png) 0 3 0 3;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetInner" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QFrame" name="frameCaption">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelCaptionLogo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string>YesPo Service Client</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="sysPanelContainer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="sysPanel" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButtonClose">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>35</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>9</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="panel">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="labelContainer" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelText">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="buttonContainer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonOK">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonYes">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Yes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonNo">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>21</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>No</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ui.qrc"/>
  <include location="../phone/phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>ns::PromptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonYes</sender>
   <signal>clicked()</signal>
   <receiver>ns::PromptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNo</sender>
   <signal>clicked()</signal>
   <receiver>ns::PromptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
